proof/refine/ARM_HYP/Interrupt_R.thy

```diff
@@ -1182,10 +1182,10 @@ lemma vgicMaintenance_invs'[wp]:
              apply (clarsimp simp: st_tcb_at'_def obj_at'_def runnable'_eq)
              apply (rule conjI)
               apply (fastforce elim!: st_tcb_ex_cap'' simp: valid_state'_def valid_pspace'_def)
-             apply (clarsimp simp: invs'_def valid_state'_def valid_idle'_def)
-             apply (clarsimp simp: st_tcb_at'_def obj_at'_def runnable'_eq pred_tcb_at'_def)
+             apply (clarsimp simp: invs'_def valid_state'_def valid_idle'_def obj_at'_def idle_tcb'_def)
             apply wps
             apply (wpsimp simp: if_apply_def2 wp: hoare_vcg_const_imp_lift hoare_drop_imps
+hoare_vcg_ex_lift hoare_vcg_conj_lift
```

Comment:
these looks like they might be here by accident (indent, but also they shouldn't be effective in `simp`)